Example/CustomActionControllers/Spotify.swift

```diff
@@ -137,35 +137,69 @@ open class SpotifyHeaderView: UICollectionReusableView {
     }
 }
 
-open class SpotifyActionController: ActionController<SpotifyCell, ActionData, SpotifyHeaderView, SpotifyHeaderData, UICollectionReusableView, Void> {
+open class SpotifyActionController: ActionController<SpotifyCell, ActionData, SpotifyHeaderView, SpotifyHeaderData, UICollectionReusableView, Void, UICollectionReusableView> {
     
     fileprivate lazy var blurView: UIVisualEffectView = {
         let blurView = UIVisualEffectView(effect: UIBlurEffect(style: .dark))
         blurView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
         return blurView
     }()
     
+    var cancelView: UIView?
+    
     open override func viewDidLoad() {
         super.viewDidLoad()
+        
         backgroundView.addSubview(blurView)
+        cancelView = UIView(frame: CGRect(x: 0, y: view.bounds.size.height - 60, width: UIScreen.main.bounds.width, height: 60))
```

Comment:
looks like the value assigned to `cancelView` here is not used later and it can be deleted.